### PR TITLE
Add local nightwatch json

### PR DESCRIPTION
**Pull request by leggsimon**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -52,4 +52,4 @@ clean-deploy: clean install build-production deploy
 
 smoke:
 	nbt test-urls ${TEST_HOST}
-	export TEST_URL=${TEST_URL}; nbt nightwatch test/browser/tests/*
+	export TEST_URL=${TEST_URL}; nbt nightwatch test/browser/tests/* --config ./nightwatch.json --env chromeLatest,firefoxLatest,ie11,ie10,ie9
```
Comment (matthew-andrews) [inline Makefile]: what's the latest story on ios/android?

Comment (leggsimon) [inline Makefile]: They are still being fixed, Tanya will let me know when they should be working.

**Comment by matthew-andrews** [comment]
Sorry to be a pedant but would you mind tidying up the whitespace in the `nightwatch.json` file … i think we normally indent json with 2 spaces but it doesn't really matter as long as it's consistent :-)

**Comment by matthew-andrews** [comment]
Otherwise :+1:

**Pull request merged**